salsa_dancing_molecules/startup/startup_script.py

```diff
@@ -0,0 +1,28 @@
+"""Program for handling startup tasks."""
+from .prepare_workspace import do_preparations
+from .use_custom_materials import materials_to_pickles
+from .config_module import read_configuration
+from .generate_json import convert_to_json
+import os
+
+
+def start(conf_path):
+    """Run the different modules of the startup program.
+
+    Args:
+        args - argument object from argparse.
+    """
+    config = read_configuration(conf_path)
+    for simulation_conf in config.values():
+        work_path = simulation_conf["workspace_path"]
+        custom_materials = simulation_conf["materials_path"]
+        material_names = simulation_conf["material"]
+        do_preparations(work_path)
+        if custom_materials.endswith('.py'):
+            materials_to_pickles(custom_materials, work_path, material_names)
+        elif '.' not in custom_materials:
```

Comment:
I do not understand what this test does? Why are you checking whether the string contains a full stop?